lib/src/components/NxSearchDropdown/__tests__/NxSearchDropdown.test.tsx

```diff
@@ -4,225 +4,203 @@
-    expect(getShallow({ searchText: 'foo', disabled: false })).toContainMatchingElement(NxDropdownMenu);
-    expect(getShallow({ searchText: 'foo', disabled: undefined })).toContainMatchingElement(NxDropdownMenu);
-    expect(getShallow({ searchText: 'foo', disabled: null })).toContainMatchingElement(NxDropdownMenu);
-    expect(getShallow({ searchText: 'foo' })).toContainMatchingElement(NxDropdownMenu);
+  it('renders dropdown with role="menu" if there are matching elements', function() {
+    const el = quickRender({ searchText: 'foo', matches: [{ id: '1', displayName: '1' }]});
+
+    expect(el.getByRole('menu')).toBeInTheDocument();
   });
 
   it('sets aria-hidden to true on the dropdown menu if the searchText is empty or disabled is true', function() {
-    expect(getShallow().find(NxDropdownMenu)).toHaveProp('aria-hidden', true);
-    expect(getShallow({ searchText: 'foo', disabled: true }).find(NxDropdownMenu)).toHaveProp('aria-hidden', true);
-    expect(getShallow({ searchText: 'foo', disabled: false }).find(NxDropdownMenu)).toHaveProp('aria-hidden', false);
-    expect(getShallow({ searchText: 'foo', disabled: undefined }).find(NxDropdownMenu))
-        .toHaveProp('aria-hidden', false);
-    expect(getShallow({ searchText: 'foo', disabled: null }).find(NxDropdownMenu)).toHaveProp('aria-hidden', false);
-    expect(getShallow({ searchText: 'foo' }).find(NxDropdownMenu)).toHaveProp('aria-hidden', false);
+    expect(quickRender().getByRole('alert', { hidden: true })).toHaveAttribute('aria-hidden', 'true');
+    expect(quickRender({ searchText: 'foo', disabled: true }).getByRole('alert', { hidden: true }))
+        .toHaveAttribute('aria-hidden', 'true');
+    expect(quickRender({ searchText: 'foo', disabled: false }).getByRole('alert'))
+        .toHaveAttribute('aria-hidden', 'false');
+    expect(quickRender({ searchText: 'foo', disabled: undefined }).getByRole('alert'))
+        .toHaveAttribute('aria-hidden', 'false');
+    expect(quickRender({ searchText: 'foo', disabled: null }).getByRole('alert'))
+        .toHaveAttribute('aria-hidden', 'false');
+    expect(quickRender({ searchText: 'foo' }).getByRole('alert'))
+        .toHaveAttribute('aria-hidden', 'false');
   });
 
   it('sets aria-live on the dropdown menu to "polite"', function() {
-    expect(getShallow().find(NxDropdownMenu)).toHaveProp('aria-live', 'polite');
+    expect(quickRender().getByRole('alert', { hidden: true })).toHaveAttribute('aria-live', 'polite');
   });
 
   it('sets aria-busy on the dropdown menu if loading is true', function() {
-    expect(getShallow().find(NxDropdownMenu)).toHaveProp('aria-busy', false);
-    expect(getShallow({ loading: true }).find(NxDropdownMenu)).toHaveProp('aria-busy', true);
+    expect(quickRender().getByRole('alert', { hidden: true })).toHaveAttribute('aria-busy', 'false');
+    expect(quickRender({ loading: true }).getByRole('alert', { hidden: true })).toHaveAttribute('aria-busy', 'true');
```

Comment:
Cleaner to construct a case where the menu isn't hidden.  Whether it's busy or not while it is hidden doesn't really matter to anything.

Reply:
> Cleaner to construct a case where the menu isn't hidden. Whether it's busy or not while it is hidden doesn't really matter to anything.

Changed here: 845fc1b3735150e5436913ee828c14e482d1c901